tests/console/perl_bootloader.pm

```diff
@@ -15,24 +14,49 @@ use strict;
 use warnings;
 use utils 'zypper_call';
 use power_action_utils 'power_action';
-use version_utils 'is_sle';
+use version_utils qw(is_sle is_opensuse);
 
 sub run {
     my ($self) = @_;
     select_serial_terminal;
+    zypper_call 'in perl-Bootloader';
 
-    if (script_run 'rpm -q perl-Bootloader' == 1) {
-        zypper_call 'in perl-Bootloader';
+    # pbl --loader is not available on <15-SP3
+    if (is_sle(">=15-SP3") || is_opensuse) {
+        if (get_var('UEFI')) {
+            assert_script_run 'pbl --loader grub2-efi';
+            validate_script_output 'cat /etc/sysconfig/bootloader | grep "^LOADER_TYPE=\"grub2-efi\"$"', sub { m/^LOADER_TYPE="grub2-efi"$/ };
+        }
+        else {
+            assert_script_run 'pbl --loader grub2';
+            validate_script_output 'cat /etc/sysconfig/bootloader | grep "^LOADER_TYPE=\"grub2\"$"', sub { m/^LOADER_TYPE="grub2"$/ };
```

Comment:
you dont need the grep part as you are already validating the content of the file